packages/apis/multisend/scripts/createPayload.js

```diff
@@ -0,0 +1,20 @@
+const { ethers } = require("hardhat");
+const fs = require("fs");
```

Comment:
What's the use of these scripts? If it's for just testing then we should create jest tests instead